app/uk/gov/hmrc/eoricommoncomponent/frontend/views/migration/match_utr_subscription.scala.html

```diff
@@ -35,33 +35,13 @@
     }
 }
 
-@intro = @{
-        if(organisationType == CdsOrganisationType.SoleTraderId | organisationType == CdsOrganisationType.IndividualId) messages("cds.matching.row-sole-trader-individual.utr.paragraph")
-    
-}
-@utrlink = @{
-    organisationType match {
-        case CdsOrganisationType.SoleTraderId | CdsOrganisationType.IndividualId => Html(messages(""))
-        case _ => hintlinkOrganisation
+@intro = {
+        @if(organisationType == CdsOrganisationType.SoleTraderId | organisationType == CdsOrganisationType.IndividualId) {
+        @Html(messages("cds.matching.row-sole-trader-individual.utr.paragraph"))
```

Comment:
Do we only show this hint text when it's a sole trader or individual? If so I think we should be passing None to the hint param below when it's neither of those otherwise it may display an empty hint <div> - can you check?